Jenkinsfile.redhat

```diff
@@ -45,66 +45,180 @@ pipeline {
             }
         }
 
-        stage('Build') {
-            steps {
-                configFileProvider([configFile(fileId: 'fuse-maven-settings', variable: 'MAVEN_SETTINGS')]) {
-                    sh "./mvnw ${MAVEN_PARAMS} clean install -Dquickly -Pmixed"
+        stage('Tests') {
+            parallel {
+                // %generated-stages-start%
+                stage('Product :: Group 01') {
+                    agent {
+                        label "rhel"
```

Comment:
Not sure whether `rhel` is the best choice here. It was the first label I saw in other Fuse pipelines.

Reply:
TBH most of the labels are meaningless and result in execution on the same agent type. The reason we have so many is probably for legacy reasons.